skunkworks_crow/src/main/res/values/strings.xml

```diff
@@ -157,4 +157,7 @@
     <string name="dev_already_connected">already connected to the device, start transferring now&#8230;</string>
     <string name="tv_form_send_success">form sent with success! \n</string>
     <string name="bluetooth_no_device_scanned">No devices are available. \n Tap refresh to search again.</string>
+    <string name="permission_location_denied">You need to get location permission to enable bluetooth feature!</string>
+    <string name="permission_open_info">Please get location permissions on the APP details page.</string>
```

Comment:
Change text to `Skunkworks-crow app requires location permission to access your Bluetooth. Please allow location permission from your app settings.`